Docs: update account-permissions (#7261)
expo/expo: docs/pages/versions/unversioned/guides/account-permissions.md

title: Account Permissions
---

> > Some features on this page are [available with a paid plan](https://expo.io/developer-services).

Reviewer: ```suggestion
> Some features on this page require [Developer Services](https://expo.io/developer-services).
```
